scripts/ci/deploy.sh

```diff
@@ -48,7 +48,7 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
   ./scripts/build/docker/run-command.sh \
     -r "$ARGV_ARCHITECTURE" \
     -s "$(pwd)" \
-    -c "make publish-aws-s3"
+    -c "make publish-aws-s3 publish-bintray-debian publish-bintray-redhat"
```

Comment:
Looks like our `installers-all` target already did exactly this. I'm going to update the deploy script to use that instead.